cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -1328,7 +1378,7 @@ func Test_buildHealthCheck(t *testing.T) {
 					},
 				},
 			},
-			errMsgPrefix: fmt.Sprintf("invalid protocol: %q specified in annotation: %q", "invalid", annDOProtocol),
```

Comment:
Looks like you replaced the test for the case of an invalid health check protocol specified? This is still possible if the user puts in an unsupported protocol in the annotation, isn't it?

Or is the Github diff view tricking me here?

Reply:
The `buildHealthCheck` method no longer does a `getProtocol` so this test case wouldn't get hit anymore.

Reply:
To clarify, this wasn't a test for invalid health check protocol, this was a test for invalid service protocol.

Reply:
Got it, missed what this test was really good for. Thanks.